ocaml/xapi/xapi_mgmt_iface.ml

```diff
@@ -48,114 +30,151 @@ let update_mh_info interface =
   in
   ()
 
-let _restart_stunnel_no_cache ~__context ~accept =
-  let (_ : Thread.t) =
-    Thread.create (fun () -> Helpers.Stunnel.restart ~__context ~accept) ()
-  in
-  ()
+module Stunnel = struct
+  let stunnel_accept_m = Mutex.create ()
```

Comment:
The names in this module don't need to include `stunnel`  as this is already provided by the module.